data/reporter.js

```diff
@@ -50,6 +50,11 @@ var initialize = function(data) {
         version.appendChild(document.createTextNode("Version "));
         version.appendChild(document.createTextNode(document.addonReports[i].version));
         td2.appendChild(version);
+
+        var e10s = document.createElement("div");
+        e10s.setAttribute("class", "version");
+        e10s.appendChild(document.createTextNode((!document.addonReports[i].multiprocessCompatible ? "Not compatible" : "Compatible") + " to multi-process."));
```

Comment:
"with multiprocess" instead "to multiprocess" reads better I think.

Also if this is just for us/add-on developers, we can probably just say e10s.